language/stdlib/modules/LBR.move

```diff
@@ -102,6 +105,17 @@ module LBR {
             Libra::currency_code<CoinType>() == Libra::currency_code<LBR>()
     }
 
+    /// We take the truncated multiplication + 1 (not ceiling!) to withdraw for each currency that makes up the `LBR`.
+    /// We do this to ensure that the reserve is always positive. We could do this with other more complex methods such as
+    /// bankers rounding, but this adds considerable arithmetic complexity.
```

Comment:
```suggestion
    /// banker's rounding, but this adds considerable arithmetic complexity.
```

Reply:
Good questions:
* I don't think this will clean those up since those use `LibraAccount::staple_lbr`. But I may be missing something?
* We can make `LBR::swap_into` private (well, we can remove it). I'll do that in this PR.